### DISCUSSION TITLE
support: Functions and Collations sintaxe

### DISCUSSION THREAD

**Discussion by edbertof** [pawelsalawa/sqlitestudio#4723] [General]
Sqlite Studio has a function editor and a collation editor, but does not provide a way for how these objects should be specified (syntax). There is no documentation to assist the user in specifying functions or collations.

**Comment by tuffnatty** [comment]
@edbertof The documentation is right there and is plentiful enough:
https://github.com/pawelsalawa/sqlitestudio/wiki/User_Manual#custom-sql-functions
https://github.com/pawelsalawa/sqlitestudio/wiki/User_Manual#custom-collations
